packages/@aws-cdk/aws-servicecatalog/lib/product-stack-history.ts

```diff
@@ -86,6 +87,34 @@ export class ProductStackHistory extends Construct {
     };
   }
 
+  /** Extracts versions from file system */
+
+  public allVersions(): CloudFormationProductVersion[] {
```

Comment:
Can you help my understand the naming choice for this? From the context I've gathered this get's all versions from a local snapshot directory. Are these always going to be "all versions"? Is that snapshot usually under version control and shared?

Maybe a name like `versionsFromSnapshot`  would be more descriptive.

Reply:
I think versionsFromSnapshot is a good name change. The original reasoning was that it collects all of the locally saved versions that were saved from the ProductStackHistory. versionsFromSnapshot is clearer.